src/pages/keyword-generator/KeywordGenerator.vue

```diff
@@ -1,27 +1,33 @@
 <script setup>
-import { ref } from 'vue'
+import { computed, ref } from 'vue'
+import { MaInput } from "@mobileaction/action-kit"
+import { MaBadge } from "@mobileaction/action-kit"
 
+const checkedNumbers = ref([])
```

Comment:
```suggestion
const selectedNGrams = ref([])
```
Changing the name from checkedNumbers to selectedNGrams might be more descriptive.